app/src/main/java/uz/direction/coroutineshomework/MemeView.kt

```diff
@@ -2,33 +2,46 @@ package uz.direction.coroutineshomework
 
 import android.content.Context
 import android.util.AttributeSet
+import android.util.Log
 import android.widget.Button
 import android.widget.ImageView
 import android.widget.TextView
 import androidx.constraintlayout.widget.ConstraintLayout
 import com.bumptech.glide.Glide
-import com.bumptech.glide.request.target.Target
+import kotlinx.coroutines.*
+
+val memeViewScope =
```

Comment:
We don't need this scope as our presenter already runs in scope, moreover you are not canceling it